server/vulnerabilities/nvd/sync/cve_syncer.go

```diff
@@ -781,15 +781,33 @@ func convertAPI20CVEToLegacy(cve nvdapi.CVE, logger log.Logger) *schema.NVDCVEFe
 	descriptions := make([]*schema.CVEJSON40LangString, 0, len(cve.Descriptions))
 	for _, description := range cve.Descriptions {
 		// Keep only english descriptions to match the legacy.
-		if description.Lang != "en" {
+		var lang string
+		switch {
+		case description.Lang == "en":
+			lang = description.Lang
+		case description.Lang == "en-US":
+			// This occurred starting with Microsoft CVE-2024-38200
+			lang = "en"
+		case strings.HasPrefix(description.Lang, "en-"):
+			lang = description.Lang
```

Comment:
Do we need to coerce this to `en` as well?

Reply:
and/or log (warn-level?) when we hit the prefix so we know when we see a new language code beyond just `en-US`

Reply:
Maybe the easiest is to drop it for now. Seems silly to write code for a case that doesn't exist. We'll fall back to empty description if no `en`/`en-US` languages are present.

Reply:
Works for me.